virttest/utils_misc.py

```diff
@@ -312,6 +312,28 @@ def find_substring(string, pattern1, pattern2=None):
     return ret[0]
 
 
+def get_match_count(check_path, key_message):
```

Comment:
Put this function into utils_logfile.py, thanks.

Reply:
THanks done